packages/react/src/input/Input.tsx

```diff
@@ -43,7 +51,34 @@ export const Input = forwardRef<HTMLInputElement, InputProps>(
         data-invalid={isInvalid}
         {...sprinkleProps}
       >
-        <input placeholder={placeholder} ref={ref} type={type} {...restProps} />
+        {leftSection && (
+          <Box flex="none" mr="8">
+            {leftSection}
+          </Box>
+        )}
+        <Box
+          asChild
+          className={clsx(
+            styles.input({
+              variant,
+            }),
+          )}
+        >
+          <input
+            defaultValue={defaultValue}
+            id={id}
+            placeholder={placeholder}
```

Comment:
are we actually doing anything with the `placeholder`? if not let's just forward it via `restProps` instead of picking it out and passing it explicitly

Reply:
Before restProps were introduced, I needed to pass the prop explicitly for the placeholder. With restProps, we wouldn't need it anymore vai.